public/js/maps.js

```diff
@@ -0,0 +1,53 @@
+const HOUSTON_COORDS = { lat: 29.7604, lng: -95.3698 }
+
+var map;
+var bounds;
+var openMarkerWindow;
+
+/*
+* Initialize the Google Map to be centerd in Houston and to add 
+* initial organization markers.
+*/
+function initMap() {
+  map = new google.maps.Map(document.getElementById("map"), {
+    center: HOUSTON_COORDS,
+    zoom: 13,
+    disableDefaultUI: true,
+    zoomControl: true,
+    styles: [
+      {
+        featureType: "poi",
+        stylers: [{ visibility: "off" }]
+      }
+    ]
+  });
+  bounds = new google.maps.LatLngBounds(HOUSTON_COORDS);
+  generateMarkers(organizations);
+}
+
+/*
+* Add markers to the map based on the data in the data array passed in. 
+*/
```

Comment:
```suggestion
/**
 * Add markers to the map based on the data in the data array passed in. 
 */
```
When you include documentation for `data` parameter use: [https://google.github.io/styleguide/jsguide.html#jsdoc-general-form](https://google.github.io/styleguide/jsguide.html#jsdoc-general-form)